### PR TITLE
013 authorisation

### PR DESCRIPTION
**Pull request by joel-and**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
     "include": [
         "src/**/*"
-    ],
+, "test/unit/middleware/AuthMiddlewareTest.ts"    ],
```
Comment (shaunganley) [inline tsconfig.json]: why is this needed?

**Review by shaunganley — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (joel-and) [reply tsconfig.json]: fixed now

**Review by anne-moore — APPROVED** [review]
